### PR TITLE
Code optimizations

### PR DESCRIPTION
**Pull request by iksaku**
Well, the title says it all :P
I have made some changes to the code, so it is more logic and doesn't need to "repeat patterns" (Like at every subcommand ``__construct()``)

Fell free to tell me what to change in order to improve this ;)

BTW: Glad to see you back (coding) @wiez :3

### CODE REVIEW

**Comment by Exxarion** [comment]
This is some great work you have done! I will merge it with the Dev branch to test it out

**Comment by wiez** [comment]
Great work my friend!

File: src/MyPlot/Commands.php [hunk 1]
```diff
@@ -48,7 +48,7 @@ public function __construct(MyPlot $plugin) {
 
     private function loadSubCommand(Subcommand $command) {
         $this->commandObjects[] = $command;
-        $commandId = count($this->commandObjects) - 1;
+        $commandId = count($this->commandObjects) + 1;
```
Comment (iJoshuaHD) [inline src/MyPlot/Commands.php]: why? o.O

Comment (wiez) [inline src/MyPlot/Commands.php]: Me hitting the wrong buttons -_-

Comment (wiez) [inline src/MyPlot/Commands.php]: Oh wait... you are right this was good the way it was

Comment (PEMapModder) [inline src/MyPlot/Commands.php]: Why do you need it at all? Shouldn't it be the count directly?

Comment (wiez) [inline src/MyPlot/Commands.php]: Well actually it doesn't really matter, but I like the first command to have id 0 :)

**Pull request merged**